components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -108,6 +109,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
     _displayMaskLayer = [[MDCFeatureHighlightLayer alloc] init];
     _displayMaskLayer.fillColor = [UIColor whiteColor].CGColor;
 
+    _accessibilityView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 1, 1)];
```

Comment:
Should this frame be the entire feature highlight in some manner?

Reply:
it is with it's accessibility frame. I didn't want it to cause the entire tappable area for non voiceover users to become dismissable and thus affect them